gekitchen/erd/registry.py

```diff
@@ -0,0 +1,85 @@
+from typing import TypedDict
+
+from .codes import ErdCode
+from .converters import *
+
+class ConverterRegistry(TypedDict):
+    key: ErdCode
+    value: ErdValueConverter
+
+_registry = ConverterRegistry({
```

Comment:
This should be renamed to not have a leading underscore since it's being imported at https://github.com/ajmarks/gekitchen/pull/7/files#diff-6268e47bd3fc9c879628ce37769b861060990bf078a59827fac63ba6710ff5b2R7